website/client/components/shops/market/index.vue

```diff
@@ -532,9 +532,11 @@ export default {
           }
 
           categories.map((category) => {
-            this.$set(this.viewOptions, category.identifier, {
-              selected: true,
-            });
+            if (!this.viewOptions[category.identifier]) {
```

Comment:
why this change?

Reply:
on pinning items the store data refreshes, including the `market.categories` which then refreshes the `categories`, this fix is to prevent the viewOptions to be reseted on that refresh (if you already unchecked one of the filters)